### PR TITLE
Bring over openshift-migration-plugin to a unified repo

### PR DESCRIPTION
**Pull request by dymurray**
Do not merge until CAM team has blessed the PR.

### CODE REVIEW

**Comment by jmontleon** [comment]
This PR's not quite as scary as it first looks. There are only about 22 files changed that aren't vendor or vendor related. And 3 of those are Dockerfile, Makefile, and Readme.md. I'm willing to VISIACK it since it's been getting tested with OADP. I think merging it and getting any issues resolved is going to be the best bet.

**Review by jmontleon — APPROVED** [review]
VISIACK

File: README.md [hunk 1]
```diff
@@ -28,9 +45,287 @@ different name, run
...
+time="2020-07-29T16:19:08Z" level=info msg="[common-backup] Entering common backup plugin" backup=oadp-operator/nginx-stateless cmd=/plugins/velero-plugins logSource="/go/src/github.com/konveyor/openshift-velero-plugin/velero-plugins/common/backup.go:25" pluginName=velero-plugins
+time="2020-07-29T16:19:08Z" level=info msg="[common-backup] Entering common backup plugin" backup=oadp-operator/nginx-stateless cmd=/plugins/velero-plugins logSource="/go/src/github.com/konveyor/openshift-velero-plugin/velero-plugins/common/backup.go:25" pluginName=velero-plugins
+time="2020-07-29T16:19:08Z" level=info msg="[common-backup] Entering common backup plugin" backup=oadp-operator/nginx-stateless cmd=/plugins/velero-plugins logSource="/go/src/github.com/konveyor/openshift-velero-plugin/velero-plugins/common/backup.go:25" pluginName=velero-plugins
+```
+#### Restore Plugin
+- Set the RestoreServerVersion annotation to correct server version
+- Set the RestoreRegistryHostname annotation to the corect hostname
```
Comment (sseago) [inline README.md]: minor typo: "correct"

**Review by sseago — APPROVED** [review]
ACK, but with a minor doc typo

**Pull request merged**